Modules/Server/Sources/PocketCastsServer/Public/ServerPodcastManager+Update.swift

```diff
@@ -104,8 +108,15 @@ extension ServerPodcastManager {
                 continue
             }
 
-            // for subscribed podcasts, only add older episodes, newer ones are handled by a refresh
```

Comment:
At first, I wasn't sure why that, but now it's clear: the refresh handles auto-download, auto-add to Up Next, etc. That's why we shouldn't update these episodes here.